components/user-header.js

```diff
@@ -180,8 +184,35 @@ function HeaderNym ({ user, isMe }) {
     : <NymView user={user} isMe={isMe} setEditting={setEditting} />
 }
 
+function SocialLink ({ name, id }) {
+  if (name === 'Nostr') {
+    const npub = hexToBech32(id)
+    return (
+      <Link className={styles.social} target='_blank' href={`https://snort.social/${npub}`} rel='noreferrer'>
+        <NostrIcon width={20} height={20} className='me-1' />
+        {npub.slice(0, 20)}...
```

Comment:
I think it's more common to show both ends for such strings. So I would suggest to use first ten characters and last ten characters.

```suggestion
        {npub.slice(0, 10)}...{npub.slice(-10)}
```